### PR TITLE
Add HPE FlexVolume Driver Chart

### PR DESCRIPTION
**Pull request by shivamerla**

### CODE REVIEW

File: charts/hpe-flexvolume-driver/v3.0.0/questions.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+categories:
```
Comment (guangbochen) [inline charts/hpe-flexvolume-driver/v3.0.0/questions.yml]: can we please add the following labels to the `questions.yml` for Rancher to use it, thanks
```
rancher_min_version: 2.3.0-rc1
labels:
  io.cattle.role: cluster # options are cluster/project
  io.rancher.certified: partner
```

File: charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+#### HPE Dynamic Provisioner ###
+---
+apiVersion: extensions/v1beta1
```
Comment (guangbochen) [inline charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml]: can we please update to `apps/v1` for k8s v1.16 capability support, thanks.

File: charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+kind: Deployment
+metadata:
+  name: hpe-dynamic-provisioner
+  namespace: kube-system
+  labels:
+    app: hpe-dynamic-provisioner
+spec:
```
Comment (guangbochen) [inline charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml]: matchLabels will be required for `apps/v1` version.
```
  selector:
    matchLabels:
       label1: value1
```

File: charts/hpe-flexvolume-driver/v3.0.0/Chart.yaml [hunk 4]
```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
+appVersion: "3.0"
+description: A Helm chart for installing the HPE Volume Driver for Kubernetes FlexVolume plugin
+icon: https://raw.githubusercontent.com/hpe-storage/co-deployments/master/docs/assets/hpedev.png
```
Comment (guangbochen) [inline charts/hpe-flexvolume-driver/v3.0.0/Chart.yaml]: would prefer to add the icon to the parent directory and use it via icon: file://../logo.png for air-gapped users, thanks.

**Comment by shivamerla** [comment]
@guangbochen thanks for the review, will update accordingly and submit changes.

File: charts/hpe-flexvolume-driver/v3.0.0/questions.yml (hunk 1 above)
Comment (shivamerla) [reply charts/hpe-flexvolume-driver/v3.0.0/questions.yml]: thanks, added it.

File: charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml (hunk 2 above)
Comment (shivamerla) [reply charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml]: done..

File: charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml (hunk 3 above)
Comment (shivamerla) [reply charts/hpe-flexvolume-driver/v3.0.0/templates/hpe-doryd.yaml]: done..

File: charts/hpe-flexvolume-driver/v3.0.0/Chart.yaml (hunk 4 above)
Comment (shivamerla) [reply charts/hpe-flexvolume-driver/v3.0.0/Chart.yaml]: done..thanks.

**Comment by shivamerla** [comment]
@guangbochen closing this PR and created a new one for dev branch: https://github.com/rancher/charts/pull/266

**Pull request closed without merging**